people.json

```
@@ -86,6 +86,51 @@
          ]
       ]
    },
+   
+   {
+      "name": "Dan-Dylan Lavictoire",
+      "image": "https://cdna.artstation.com/p/users/avatars/000/678/892/large/9850decab6e8f4a20895d228cd97e156.jpg?1563831406",
+      "skills": [
+         "game design",
+         "programming",
+         "production",
+         "narrative design",
+         "game audio",
+         "art",
```

Hi @DaDyDoux13! Unfortunately, strict JSON format doesn't support trailing commas, so this will cause the site to error out when loading this info - removing the comma solves the issue ✨
```suggestion
         "art"
```